ansible-deployment/switch/tasks/switches.j2

```diff
@@ -33,3 +33,29 @@ mac=e4:c7:22:c0:0b:bc
 manage_vlans=True
 ansible_connection=local
 stp_edge=True
+[ansible:MOC-R4PAC21-SW-TORS-A]
+ansible_connection=ansible.netcommon.network_cli
+ansible_network_os=dellemc.os9.os9
+ansible_host=10.2.10.1
+ansible_user=admin
+ansible_ssh_pass={{ ansible_switch_ssh_password }}
+ansible_become_password={{ ansible_switch_ssh_password }}
+ansible_become=yes
+ansible_become_method=enable
+mac=d8:9e:f3:a4:46:a2
+manage_vlans=True
+ansible_connection=local
+stp_edge=True
+[ansible:MOC-R4PAC21-SW-TORS-B]
+ansible_connection=ansible.netcommon.network_cli
+ansible_network_os=dellemc.os9.os9
+ansible_host=10.2.10.2
+ansible_user=admin
+ansible_ssh_pass={{ ansible_switch_ssh_password }}
+ansible_become_password={{ ansible_switch_ssh_password }}
```

Comment:
Are the switch passwords all the same as the one for `switch`? If not, these parameter names should probably be different.

Reply:
seems reasonable!

Reply:
Done!